app/src/utils/madlib/DisplayNames.ts

```diff
@@ -1,6 +1,6 @@
-import { VariableId } from "../../data/variableProviders";
+import { MetricId } from "../../data/variableProviders";
 
-export const VARIABLE_DISPLAY_NAMES: Record<VariableId, string> = {
+export const METRIC_DISPLAY_NAMES: Record<MetricId, string> = {
```

Comment:
Can this map be deprecated in favor of the variable configs? (obviously this is a separate PR, I just noticed)

Reply:
yeah I had the same thought.. I'll make a TODO so we dont' forget